File: ProyectoFinal/ProyectoFinal/MenuAlgoritmos.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace ProyectoFinal
{
    public partial class MenuAlgoritmos : Form
    {
        public MenuAlgoritmos()
        {
            InitializeComponent();
        }

        private void btnBurbuja_Click(object sender, EventArgs e)
        {
            AlgoritmosdeOdenamiento.Burbuja il = new AlgoritmosdeOdenamiento.Burbuja();
            il.Show();
            this.Close();
        }

        private void btnBurbujaB_Click(object sender, EventArgs e)
        {
            AlgoritmosdeOdenamiento.BurbujaBidireccional i = new AlgoritmosdeOdenamiento.BurbujaBidireccional();
            i.Show();
            this.Close();
        }

        private void btnCuentas_Click(object sender, EventArgs e)
        {
            AlgoritmosdeOdenamiento.Cuentas sam = new AlgoritmosdeOdenamiento.Cuentas();
            sam.Show();
            this.Close();
        }
    }
}

File: ProyectoFinalVB/ProyectoFinalVB/MenuAlgoritmos.vb
Public Class MenuAlgoritmos
    Private Sub btnBurbuja_Click(sender As Object, e As EventArgs) Handles btnBurbuja.Click
        Dim il As Burbuja = New Burbuja()
        il.Show()
        Me.Close()
    End Sub

    Private Sub btnBurbujaB_Click(sender As Object, e As EventArgs) Handles btnBurbujaB.Click
        Dim i As BurbujaBidireccional = New BurbujaBidireccional()
        i.Show()
        Me.Close()
    End Sub

    Private Sub btnCuentas_Click(sender As Object, e As EventArgs) Handles btnCuentas.Click
        Dim sam As Cuentas = New Cuentas()
        sam.Show()
        Me.Close()
    End Sub
End Class